xml/operations-configuring-identity-websso.xml

```diff
@@ -76,8 +85,9 @@
    assertions provided by the identity provider.
   </para>
   <para>
+   WebSSO support both SAML and OpenID methods.
    The following is the workflow that depicts how horizon and keystone supports
```

Comment:
"horizon and keystone support" (no letter s).

Reply:
Done